tests/test_plotting.py

```diff
@@ -507,7 +507,7 @@ def test_plot_boundary_surface(self):
         return fig
 
     @pytest.mark.unit
-    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
```

Comment:
I assume this change was for debugging. Can you change it back to `tol_2d`?

Reply:
The change was intentional, since the boundaries plot has much less lines than i.e. a countour plot, image plot, heatmap... so the RMS is less sensitive to change and the threshold for the test should be lowered accordingly. 

If you think this will cause confusion, I'll revert the change.